dll/win32/shell32/shlfileop.cpp

```diff
@@ -1972,14 +1974,26 @@ validate_operation(LPSHFILEOPSTRUCTW lpFileOp, FILE_LIST *flFrom, FILE_LIST *flT
                     {
                         strTitle.LoadStringW(IDS_MOVEERRORTITLE);
                         if (IsAttribDir(feFrom->attributes))
+#ifdef __REACTOS
```

Comment:
1: This is the wrong preprocessor macro
2: It makes no sense in this file.

Reply:
```suggestion
#ifdef __REACTOS__
```

Reply:
What Mark wrote: still wrong and forked module anyway.

Reply:
Fixed.